src/cf.data.pre

```diff
@@ -5211,8 +5170,8 @@ DOC_START
 	The default formats available (which do not need re-defining) are:
 
 logformat squid      %ts.%03tu %6tr %>a %Ss/%03>Hs %<st %rm %ru %[un %Sh/%<a %mt
-logformat common     %>a %[ui %[un [%tl] "%rm %ru HTTP/%rv" %>Hs %<st %Ss:%Sh
-logformat combined   %>a %[ui %[un [%tl] "%rm %ru HTTP/%rv" %>Hs %<st "%{Referer}>h" "%{User-Agent}>h" %Ss:%Sh
+logformat common     %>a - %[un [%tl] "%rm %ru HTTP/%rv" %>Hs %<st %Ss:%Sh
+logformat combined   %>a - %[un [%tl] "%rm %ru HTTP/%rv" %>Hs %<st "%{Referer}>h" "%{User-Agent}>h" %Ss:%Sh
```

Comment:
I think keeping these two built-in formats intact (as is done in the current PR code) is the lesser evil than removing the second field. This comment is not requesting any PR changes. The need for an always-dash field does raise an obvious red flag for our overall approach to maintenance of these built-in formats (which arguably have other problems as well). The text below discusses possible ways to avoiding similar problems in the future...

Long-term, we should probably deprecate and then stop providing named built-in definitions to avoid being stuck with legacy definitions (that cannot be improved without inconveniencing a lot of admins). Needless to say, we can still publish useful definitions in documentation, so that admins can copy-paste the definitions they like into their configurations (and become responsible for their maintenance).

Alternatively, we can document that built-in format details may change at any time without deprecation (even if those changes are going to break existing log consumers), and that admins are expected to copy-paste these format definitions if they want to protect themselves from such changes. This alternative feels inferior to me because it keeps Squid code more complex without reducing long-term admin load in most important cases.